src/workspaces/cookiefactoryv2/README.md

```diff
@@ -56,21 +104,59 @@ This project walks you through the process of setting up the Bakersville Cookie
       ```
     - Open scene in console: `https://us-east-1.console.aws.amazon.com/iottwinmaker/home?region=us-east-1#/workspaces/__FILL_IN__/scenes/CookieFactory`
 
-4. Setup Web Application
-   - Follow instructions in [CookieFactoryDemo](./CookieFactoryDemo/README.md)
+
+
+## Configure Web Application
+
+First change directory to CookieFactoryDemo
+``` 
+cd ../CookieFactoryDemo 
+```
+
+1. Change `WORKSPACE_ID` in `src/config/sites.ts` to your workspace ID.
+2. Use the following CLI command to administratively set the password for your demo Cognito user, Fran. Your User_Pool_id can be found after your CDK deploy completed in the CloudFormationOutput (printed to console). The demo Username is "fran@cookiefactory". Be sure to create a password that meets the default Cognito password requirements (Lowercase letter, Uppercase letter, Number, Symbol, Length >= 8)
```

Comment:
> User_Pool_id

consistency: looks like it's "UserPoolId" in the cdk output

Reply:
will update output